### ISSUE TITLE
Judgement Metric not passed through

### ISSUE THREAD

**Issue by kkarrancsu** [HDI-Project/ATM#31]
Line 341 of `atm/worker.py` does not pass the score target, so regardless of what the user chooses in `run_config.yaml`, the code determines the best classifier based on the `mu_sigma` computation.  It seems there is some inconsistency in nomenclature between the `mu_sigma` and the `cv` options. 

Was the original intent to use `mu_sigma` to correspond to the highest lower error bound, `cv` to correspond to the average `cv` score, and `test` to correspond to average test score?  `mu_sigma` is not in the keys of the database, so its not a valid `run_config.yaml` value.  If we take `cv` to mean the same thing as `mu_sigma`, then one approach to deal with this may be (starting at Line 341 of atm/worker.py):

```
        if('cv' in self.datarun.score_target):
            score_target_in = 'mu_sigma'
        else:
            score_target_in = self.datarun.score_target
        old_best = self.db.get_best_classifier(datarun_id=self.datarun.id,
                                               score_target=score_target_in)

        cur_cv_val       = model.cv_judgment_metric
        cur_cv_err       = model.cv_judgment_metric_stdev
        cur_test_val     = model.test_judgment_metric
        
        if old_best is not None:
            old_cv_val   = old_best.cv_judgment_metric
            old_cv_err   = 2*old_best.cv_judgment_metric_stdev
            old_test_val = old_best.test_judgment_metric
        if('cv' in self.datarun.score_target):
            _log('Judgment metric (%s): %.3f +- %.3f' %
                 (self.datarun.metric, cur_cv_val, cur_cv_err))
            if old_best is not None:
                if (cur_cv_val - cur_cv_err) > (old_cv_val - old_cv_err):
                    _log('New best score! Previous best (classifier %s): %.3f +- %.3f' %
                         (old_best.id, old_cv_val, old_cv_err))
                else:
                    _log('Best so far (classifier %s): %.3f +- %.3f' %
                         (old_best.id, old_cv_val, old_cv_err))
        else:
            _log('Judgment metric (%s): %.3f' %
                 (self.datarun.metric, cur_test_val))
            if old_best is not None:
                if (cur_test_val) > (old_test_val):
                    _log('New best score! Previous best (classifier %s): %.3f' %
                         (old_best.id, old_test_val))
                else:
                    _log('Best so far (classifier %s): %.3f' %
                         (old_best.id, old_test_val))
```

**Comment by bcyphers** [comment]
Good catch. 

There is definitely confusion elsewhere in the code -- mu_sigma should be a `score_target`, not a `metric`, as it's a way of using the cv_judgment_metric and its standard deviation to compute a lower confidence bound. There are other inconsistencies between the metrics used in `metrics.py` and the ones that ATM can actually be configured to use. I'm going to address all that (and this) in another pull request. I'll edit this comment once I push it.

I should point out that this isn't actually the code that chooses which classifier to try next -- that happens in [btb](https://github.com/HDI-Project/BTB), and scores are passed to BTB in Worker.select_hyperpartition() (the relevant line is 244). Still, this is confusing to users, as the output does not reflect the way ATM is working.

**Comment by bcyphers** [comment]
Closed by #34
